### DISCUSSION TITLE
Not able to install using the composer dkan on drupal 9

### DISCUSSION THREAD

**Discussion by rtnilesh** [GetDKAN/dkan#3829] [Q&A]
please help here how to install dkan 2 in drupal 9  via docker it is not working

**Comment by dafeder** [comment]
HI @rtnilesh can you explain in a bit more detail where it's failing, and what kind of environment you're working in? Thanks.

**Reply by rtnilesh** [reply]
Hi @dafeder Thanks for your response , I tried to install in local as well as in ubuntu server using the dktl commands it is installing the drupal and while installing the front end it is falling wih npm saas error and if some how while degrading the version of node i am installing then front end is not building , Help required here to install DKAN and drupal ,DKAN front end. please, am waiting for your response.

**Reply by dafeder** [reply]
@rtnilesh can you share the commands you are using to install the frontend and the full error you're getting? cc @dgading

**Reply by rtnilesh** [reply]
Hi @dafeder  and @dgading thanks for your response, now again am installing freshly on ubuntu, using dktl command and following this https://dkan.readthedocs.io/en/latest/installation/local-dev.html but getting error like dktl command not found...help required here.
![image](https://user-images.githubusercontent.com/108806560/188200909-34968ad1-6c55-498a-8ca2-ac87a41c7698.png) ...also please if there is any video tutorial or any document with i need to follow,..for installation. coz every time am getting error. step by step guide required here..please
